### ISSUE TITLE
Change licence?

### ISSUE THREAD

**Issue by mafooo** [emreakay/CodeIgniter-Aauth#88]
Have you considered changing the licence to MIT, or dual licence - to bring it inline with codeigniter 3's licence.

GPL and derivatives scare off a log of people who might use the library, and the more people who use it - the more are likely to contribute back.

**Comment by ManeBit** [comment]
@emreakay Any thoughts on this?

**Comment by emreakay** [comment]
i may consider this. 
i want to know everone's thougth before i decide

**Comment by REJack** [comment]
i would choose the MIT licence, we need a poll for it.

**Comment by xocialize** [comment]
I would agree that the license should be MIT to match codeigniter 3

**Comment by BamaStangGuy** [comment]
Agreed. Really good work with this. Would love to see more contributors.

**Comment by krishnaguragain** [comment]
Mit

**Comment by emreakay** [comment]
Ok. We are going to change to MIT.

**Comment by REJack** [comment]
Like @emreakay said,

we going to MIT with v3.0.0.

**Comment by Poncho09** [comment]
ADPL

**Comment by REJack** [comment]
@pars0097 what?

**Comment by Poncho09** [comment]
ADPL is a new license for the software free.
